pkgs/by-name/bi/bitwarden-cli/package.nix

```diff
@@ -28,7 +28,7 @@ buildNpmPackage rec {
 
   nodejs = nodejs_20;
 
-  npmDepsHash = "sha256-L7/frKCNlq0xr6T+aSqyEQ44yrIXwcpdU/djrhCJNNk=";
+  npmDepsHash = "sha256-YzhCyNMvfXGmgOpl3qWj1Pqd1hY8CJ9QLwQds5ZMnqg=";
 
   nativeBuildInputs = [
     (python3.withPackages (ps: with ps; [ setuptools ]))
```

Comment:
`distutils` was required by https://github.com/electron/rebuild/issues/1116, from this release it builds without that and even without python3
```suggestion
```

Reply:
> even without python3

Why's that? Do they ship a Python binary?

Reply:
Looks like they aren't even using `node-gyp`.

WIth the 2024.11.0 release, putting `rm -r node_modules/@electron/node-gyp/configure.js` [before the `npm rebuild` command](https://github.com/NixOS/nixpkgs/blob/f5a231b9ef276ce0d5f4cd37834ac0caf9fab62a/pkgs/build-support/node/build-npm-package/hooks/npm-config-hook.sh#L104), bitwarden-cli still builds. Just to make sure doing the same in the older 2024.9.0 version with `rm -r node_modules/node-gyp/configure.js` (electron hasn't forked node-gyp in that time) the build fails as expected with it not finding the configure script.

However the strange thing is I can delete the whole `node-gyp` directory in both 2024.9.0 and 2024.11.0 releases and the build still passes, is it possible that it wasn't even used before?